Update setup scripts domain to pull Redpanda packages from the new Artifact Registry (#1749)
redpanda-data/documentation: docs/deploy/deployment-option/cloud/managed-connectors/monitor-connectors.mdx

---

> title: Connectors monitoring

Reviewer: ```suggestion
title: Monitor Connectors
```

---

<head>
    <meta name="title" content="Connectors monitoring | Redpanda Docs"/>
    <meta name="description" content="Monitor Managed Connectors with metrics."/>
    <link rel="canonical" href="https://docs.redpanda.com/docs/deploy/deployment-option/cloud/managed-connectors/monitor-connectors/" />
</head>

> Redpanda exports metrics for Managed Connectors through a Prometheus HTTPS endpoint for you to monitor connectors health.
> You can use Prometheus and Grafana to visualize the metrics and set up alerting.

Reviewer: ```suggestion
You can monitor connector health using the managed connector metrics, which Redpanda exports through a Prometheus HTTPS endpoint. You can use Prometheus and Grafana to visualize the metrics and set up alerts.
```

## Configure Prometheus

[Prometheus](https://prometheus.io/) is a system monitoring and alerting tool. It collects and stores metrics as time-series data identified by a metric name and key/value pairs.

:::note

> You can quickly start by [running the Prometheus and Grafana locally](https://github.com/redpanda-data/observability/tree/main/cloud).

Reviewer: ```suggestion
You can quickly start by [running Prometheus and Grafana locally](https://github.com/redpanda-data/observability/tree/main/cloud).
```
